doc/Glossary/index.md

```diff
@@ -6,74 +6,232 @@ nav: /:path/Glossary/
-| Transport channel                                       | A Kaa client component responsible for communication with a Kaa server. Refer to Transports for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
```

Comment:
please remove "(also, Admin UI and web UI)" Only one name in glossary per subject.